pkg/generate/sort.go

```diff
@@ -0,0 +1,86 @@
+package generate
+
+import (
+	"fmt"
+
+	"github.com/openebs/api/v2/pkg/apis/openebs.io/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// DeviceList is a LinkedList of BlockDevices
+type DeviceList struct {
+	item v1alpha1.BlockDevice
```

Comment:
nit: If we made it as pointer copying of things will be reduced to great extent as compared to struct copy